README.md

```diff
@@ -11,6 +11,7 @@ Additional OGC API - Feature providers are listed below, along with a matrix of
 | `CKAN`             | ✅/✅                    | results/hits | ❌   | ❌       | ✅      | ✅            | ❌  | ❌           | ✅  |
 | `PsuedoPostgreSQL` | ✅/✅                    | results/hits | ✅   | ✅       | ✅      | ✅            | ✅  | ❌           | ✅  |
 | `SPARQL`           | ❌/✅                    | results/hits | ❌   | ❌       | ❌      | ❌            | ❌  | ❌           | ❌  |
+| `GeoPandas`        | ✅/✅                    | results/hits | ✅   | ✅       | ✅      | ✅            | ❌  | ❌           | ❌  |
```

Comment:
It looks like you added transactions? And you should add CRS (which is just a simple decorator function (see https://github.com/geopython/pygeoapi/pull/1729/files)